### PR TITLE
Add a sample values.yaml for amko

### PR DESCRIPTION
**Pull request by hemantavi**

### CODE REVIEW

File: charts/incubator/amko/values.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Default values for amko.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: 10.79.172.11:5000/avi-buildops/amko
```
Comment (sudswasavi) [inline charts/incubator/amko/values.yaml:8]: pls change this to the public registry.

File: charts/incubator/amko/values.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  gslbLeaderController: ""
+  controllerVersion: "18.2.9"
+  memberClusters:
+    - clusterContext: "cluster1-admin"
+    - clusterContext: "cluster2-admin"
+  refreshInterval: 1800
+  logLevel: "INFO"
+
+gslbLeaderCredentials:
+  username: "admin"
```
Comment (sudswasavi) [inline charts/incubator/amko/values.yaml:21]: pls remove these and keep it blank.

File: charts/incubator/amko/values.yaml (hunk 1 above)
Comment (hemantavi) [reply charts/incubator/amko/values.yaml]: Done.

File: charts/incubator/amko/values.yaml (hunk 2 above)
Comment (hemantavi) [reply charts/incubator/amko/values.yaml]: Done.

**Pull request merged**